mini-graph-lib.js

```diff
@@ -8,10 +8,11 @@ function getValueArray(items) {
   return items.map(item => Number(item.state) || 0);
 }
 
-function calcCoordinates(values, width, height, line_width) {
+function calcCoordinates(invalues, width, height, line_width) {
   const margin = line_width;
   width -= margin * 2;
   height -= margin * 2;
+  let values = invalues.filter(Boolean)
```

Comment:
Although this seems to work in most cases, it does seem to also filter out values assigned zero (0).

Reply:
`let values = invalues.filter(x => !isNaN(x));`
would do it. But maybe its not so good to filter this out as it might change the XRation scaling when a point is missing?
Maybe use
```
  let filtered = values.filter(x => !isNaN(x));
  const min = (Math.min.apply(null, filtered) * 0.99); // min will get NaN if any value is NaN when not filtered here
  const max = (Math.max.apply(null, filtered) * 1.01); // max will get NaN if any value is NaN when not filtered here
```
and then remove the point in the getPath when it has NaN